src/lasuite/malware_detection/backends/jcop.py

```diff
@@ -0,0 +1,222 @@
+"""Module contains the JCOP backends for the malware detection system."""
+
+import hashlib
+import logging
+from http import HTTPStatus
+
+import requests
+from celery import shared_task
+from django.conf import settings
+from django.core.files.storage import default_storage
+from django.utils.module_loading import import_string
+from requests_toolbelt.multipart.encoder import MultipartEncoder
+
+from lasuite.tools.settings import setting_name
+
+from ..enums import ReportStatus
+from ..exceptions import MalwareDetectionInvalidAuthenticationError
+from .base import BaseBackend
+
+logger = logging.getLogger(__name__)
+
+
+class JCOPBackend(BaseBackend):
+    """A backend that uses JCOP to detect malware."""
+
+    def analyse_file(self, file_path: str, **kwargs) -> None:
+        """Trigger a process to analyse a file using JCOP service."""
+        if not default_storage.exists(file_path):
+            raise FileNotFoundError(f"File {file_path} not found")
+
+        return analyse_file_async.delay(file_path, **kwargs)
```

Comment:
```suggestion
        analyse_file_async.delay(file_path, **kwargs)
```
From the base class, you should return `None` (not `AsyncResult`)